src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # BinSkim Release History
 
+## Unreleased
+
+* FEATURE: support analyzing binaries build with Clang when command line was recorded. [#548](https://github.com/microsoft/binskim/pull/548)
```

Comment:
nit: Capitalize `Support` to match the rest of the file.

Reply:
agree, fixed